uniffi/src/bindings/python/templates/RustBufferHelper.py

```diff
@@ -46,6 +46,12 @@ def getInt(self):
     def putInt(self, v):
         self._pack_into(4, ">i", v)
 
+    def getLong(self):
+        return self._unpack_from(4, ">Q")
+
+    def putInt(self, v):
```

Comment:
```suggestion
    def putLong(self, v):
```

Not a big deal since it's Python 😶